.gitignore

```diff
@@ -4,3 +4,5 @@
 tests/last-fails
 tests/last-run.log
 .cargo
+.idea/
+*.iml
```

Comment:
Could you add them to your global gitignore instead? Accepting any IDEs makes our gitignore fat and hard to maintain.